Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
The option replaces a multi-step procedure with a single-step procedure.

```
brew install --ignore-installed-specs godep
```

vs.

```
brew unlink go
brew install go
brew install godep
```

So we don't _need_ the option if the multi-step procedure is an acceptable UX.